applications/common/backend/src/main/java/pl/ds/kyanite/common/components/models/table/TableCellComponent.java

```diff
@@ -16,39 +16,137 @@
 
 package pl.ds.kyanite.common.components.models.table;
 
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.StreamSupport;
 import javax.annotation.PostConstruct;
 import javax.inject.Inject;
 import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
 import org.apache.sling.api.resource.Resource;
-import org.apache.sling.models.annotations.Default;
 import org.apache.sling.models.annotations.DefaultInjectionStrategy;
 import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.injectorspecific.SlingObject;
+import pl.ds.kyanite.common.components.models.table.dto.TableCellData;
+import pl.ds.kyanite.common.components.models.table.dto.TableRowData;
 
+@Slf4j
 @Model(adaptables = Resource.class, defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL)
 public class TableCellComponent {
```

Comment:
Worth to note - previous (underlying) PR make the Table**Head**CellComponent extend from TableCellComponent (they had identical - but duplicated - code before)